cli/switch.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
This is the new functionality for using the Bastion account, which is a workaround due to technical limitations in AD if I remember correctly. The purpose of this command is to allow users to retrieve temporary session credentials for a "bastion" account, with which they can then "assume role" into any other account.